src/web/background.py

```diff
@@ -136,7 +145,13 @@ def download_and_preprocess(path_output_folder, url=default_url):
     try:
         dataset = download_as_dataframe(url)
     except:
-        print(f"URL is not aviable:\n{default_url}")
+        dataset = None
+    
+    # Cancel if data is not aviable
+    if dataset is None:
+        print("Something went wrong when downloading the new data")
+        print("Cancelling update")
+        return
```

Comment:
lo mismo aquí, `None` p `raise Error`. en cualquier caso, `return` creo que cumpla PEP8

```suggestion
        return None
```

Reply:
Decidir si devolver None o levantar un Error